apps/fluent-tester/src/FluentTester/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { ExperimentalCheckboxTestPageId } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { Checkbox } from '@fluentui-react-native/experimental-checkbox';
+import { useTheme } from '@fluentui-react-native/theme-types';
+import { View, TextInput, TextStyle } from 'react-native';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+
+function onChangeUncontrolled(isChecked: boolean) {
+  console.log(isChecked);
```

Comment:
Is this meant to be checked in?

Reply:
the console.log line? it appears in our other checkbox test page so I was planning to check it in